frontend/src/components/Footer/index.js

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
 import Box from '@material-ui/core/Box';
+import { Grid, makeStyles, Typography } from '@material-ui/core';
 
-const Footer = () => <Box>Work in progress for Footer</Box>;
+const useStyle = makeStyles((theme) => ({
+  footer: {
+    height: '45px',
```

Comment:
Use theme.spacing instead of hardcoded values

Reply:
Done